OBSDOCS-689: add observability dir and update topic map (#71248)
openshift/openshift-docs: observability/index.adoc

:_mod-docs-content-type: ASSEMBLY
[id="observability-overview"]
= {product-title} Observability overview
include::_attributes/common-attributes.adoc[]
:context: observability-overview

toc::[]

> {product-title} Observability provides real-time visibility, monitoring, and analysis of various system metrics, logs, traces, and events to help users quickly diagnose and troubleshoot issues before they impact systems or applications. To help ensure the reliability, performance, and security of your applications and infrastructure, {product-title} offers the following Observability components:

Reviewer: Is "OpenShift Container Platform Observability" the official name? It might be better to check that and maybe create an attribute for whatever the name is to make sure it's used consistently

Author: It's supposed to be Red Hat OpenShift Observability. Updated with an attribute.

* Monitoring
* Logging 
* Distributed tracing
* Red Hat build of OpenTelemetry
* Network Observability

> * Power monitoring

Reviewer: Probably better to use attributes here too, where they exist

> OpenShift Observability connects the open source Observability operators across components to create a unified observability experience. The components of OpenShift Observability work together to help you collect, store, deliver, analyze, and visualize data.

Reviewer: Operators should always be capitalized

> == Monitoring

Reviewer: add IDs for headings

> Monitor the health and performance of your applications running on OpenShift with metrics and customized alerts for CPU, memory, and network resources. Deploy and manage in-cluster monitoring stack components with the Cluster Monitoring Operator.

Reviewer: OpenShift shouldn't be spelled out on its own, the product attribute should always be used instead

For more information, see xref:../distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].
//after the file is added to the observability directory, update xref path to ../observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].

== {OTELName}
Instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior. Use open source backends like Tempo or Prometheus, or use commercial offerings. Learn a single set of APIs and conventions, and own the data that you generate. 

For more information, see xref:../otel/otel-release-notes.adoc#otel-release-notes[{OTELName} release notes]
//after the file is added to the observability directory, update xref path to ../observability/otel/otel-release-notes.adoc#otel-release-notes[{OTELName} release notes]. 

> == Network Observability

Reviewer 2: Network Observability index /lgtm Thank you @linfraze

> Observe the network traffic for OpenShift Container Platform clusters and create network flows with the Network Observability Operator. View and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting.

Reviewer: OCP shouldn't be spelled out here, use the product attribute

For more information, see xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability overview]
//after the file is added to the observability directory, update xref path to ../observability/network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability overview]. 

== Power monitoring 

> Monitor the power usage of workloads and identify the most power-consuming namespaces running in an OpenShift cluster with key power consumption metrics, such as CPU or DRAM measured at container level. Visualize energy-related system statistics with the power monitoring Operator.

Reviewer: same as above re OpenShift

:_mod-docs-content-type: ASSEMBLY
[id="observability-overview"]
= {product-title} Observability overview
include::_attributes/common-attributes.adoc[]
:context: observability-overview

toc::[]

{ObservabilityLongName} provides real-time visibility, monitoring, and analysis of various system metrics, logs, traces, and events to help users quickly diagnose and troubleshoot issues before they impact systems or applications. To help ensure the reliability, performance, and security of your applications and infrastructure, {product-title} offers the following {ObservabilityShortName} components:

* Monitoring
* {logging-uc} 
* Distributed tracing
* {OTELName}
* Network Observability
* {PM-shortname-c}

> {ObservabilityLongName} connects observability tools and technologies to create a unified {ObservabilityShortName} experience. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.

Reviewer 3: ```suggestion
{ObservabilityLongName} connects observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
```
To my ears, `solution` seems more focused on how our tech helps customers achieve goals than the word `experience`.

[id="monitoring-overview"]
== Monitoring

> Monitor the health and performance of your applications running on {product-title} with metrics and customized alerts for CPU, memory, and network resources. Deploy and manage in-cluster monitoring stack components with the Cluster Monitoring Operator.

Reviewer 3: ```suggestion
Monitor the in-cluster health and performance of your applications running on {product-title} with metrics and customized alerts for CPU and memory usage, network connectivity, and other resource usage. Monitoring stack components are deployed and managed by the Cluster Monitoring Operator.
```
- Edit for more precise description. As currently worded, it sounds like alerts and metrics are limited to those 3 categories, when they are not (for example, there are metrics and alerts related to storage as well).
- Edit so that it doesn't sound like users can deploy the CMO themselves. It's always deployed with every OCP installation, and users cannot turn it on or off and cannot deploy it themselves. They can configure and manage some options available in the CMO, but explicit configuration is not necessarily required.

:_mod-docs-content-type: ASSEMBLY
[id="observability-overview"]
= {product-title} Observability overview
include::_attributes/common-attributes.adoc[]
:context: observability-overview

toc::[]

{ObservabilityLongName} provides real-time visibility, monitoring, and analysis of various system metrics, logs, traces, and events to help users quickly diagnose and troubleshoot issues before they impact systems or applications. To help ensure the reliability, performance, and security of your applications and infrastructure, {product-title} offers the following {ObservabilityShortName} components:

* Monitoring
* {logging-uc} 
* Distributed tracing
* {OTELName}
* Network Observability
* {PM-shortname-c}

> {ObservabilityLongName} connects open source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.

Reviewer 4: When an adjective (per ISG):
```suggestion
{ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
```

Author: @eromanova97 some Power Monitoring doc updates to apply in the next merge

For more information, see xref:../logging/cluster-logging.adoc#cluster-logging[About Logging].
//after the file is added to the observability directory, update xref path to ../observability/logging/cluster-logging.adoc#cluster-logging[About Logging].

[id="distr-tracing-architecture-index"]
== Distributed tracing
Store and visualize large volumes of requests passing through distributed systems, across the whole stack of microservices, and under heavy loads. Use it for monitoring distributed transactions, gathering insights into your instrumented services, network profiling, performance and latency optimization, root cause analysis, and troubleshooting the interaction between components in modern cloud-native microservices-based applications.

For more information, see xref:../distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].
//after the file is added to the observability directory, update xref path to ../observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].

[id="otel-release-notes-index"]
== {OTELName}

> Instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior. Use open source backends like Tempo or Prometheus, or use commercial offerings. Learn a single set of APIs and conventions, and own the data that you generate. 

Reviewer 4: "open-source" when an adjective, and "back end" when a noun (per ISG):
```suggestion
Instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior. Use open-source back ends like Tempo or Prometheus, or use commercial offerings. Learn a single set of APIs and conventions, and own the data that you generate. 
```

> For more information, see xref:../otel/otel-release-notes.adoc#otel-release-notes[{OTELName} release notes]

Reviewer 2: ```suggestion
For more information, see xref:../otel/otel-release-notes.adoc#otel-release-notes[{OTELName} release notes].
```

//after the file is added to the observability directory, update xref path to ../observability/otel/otel-release-notes.adoc#otel-release-notes[{OTELName} release notes]. 

[id="network-observability-overview"]
== Network Observability
Observe the network traffic for {product-title} clusters and create network flows with the Network Observability Operator. View and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting.

> For more information, see xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability overview]

Reviewer 2: ```suggestion
For more information, see xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability overview].
```

//after the file is added to the observability directory, update xref path to ../observability/network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability overview]. 

[id="power-monitoring-overview-index"]
== Power monitoring 

> Monitor the power usage of workloads and identify the most power-consuming namespaces running in a cluster with key power consumption metrics, such as CPU or DRAM measured at container level. Visualize energy-related system statistics with the {PM-operator}.

Reviewer 4: Suggestion only:
```suggestion
Monitor the power usage of workloads and identify the most power-consuming namespaces running in a cluster with key power consumption metrics, such as CPU or DRAM measured at the container level. Visualize energy-related system statistics with the {PM-operator}.
```

> For more information, see xref:../observability/power_monitoring/power-monitoring-overview.adoc#power-monitoring-overview[Power Monitoring overview]

Reviewer 2: ```suggestion
For more information, see xref:../observability/power_monitoring/power-monitoring-overview.adoc#power-monitoring-overview[Power Monitoring overview].
```
